Core/fft_mt_r2iq.h

```diff
@@ -2,6 +2,8 @@
 
 #include "r2iq.h"
 #include "fftw3.h"
+
+// use up to this many threads - really that many? (HA)
 #define N_R2IQ_THREAD 64
```

Comment:
my laptop show 4 threads are more than enough.

Reply:
i have same opionion .. and fear some mess if operating system might change thread/core association - just some paranoia

Reply:
Yes. My old I7-3770 shows 4 cores and 8 threads limit :-)

Reply:
@ik1xpv : i'm not sure if i understood your comment correct. would you suggest `#define N_R2IQ_THREAD 4` or not?

Reply:
this is not actual thread. this is max threads. the thread number is min(64, # of cores). I suggest 4 here.

Reply:
@howard0su yes, i knew. rename N_R2IQ_THREAD to N_MAX_R2IQ_THREADS ?

Reply:
fine with me.

Reply:
renamed and reduced to 4

Reply:
> @ik1xpv : i'm not sure if i understood your comment correct. would you suggest `#define N_R2IQ_THREAD 4` or not?

> @hayguen : I made on my PC  a run with  N_MAX_R2IQ_THREADS 4, 8 or 64.
I'm not able to see difference in CPU load. It seems to me 4 is enough.